tests/src/core/testqgstiles.cpp

```diff
@@ -65,24 +67,26 @@ void TestQgsTiles::cleanupTestCase()
 void TestQgsTiles::test_matrixFromWebMercator()
 {
   QgsTileMatrix tm = QgsTileMatrix::fromWebMercator( 0 );
+  QCOMPARE( tm.crs().authid(), "EPSG:3857" );
   QCOMPARE( tm.zoomLevel(), 0 );
   QCOMPARE( tm.matrixWidth(), 1 );
   QCOMPARE( tm.matrixHeight(), 1 );
   QVERIFY( qgsDoubleNear( tm.extent().xMinimum(), -20037508.3427892, 0.0000001 ) );
   QVERIFY( qgsDoubleNear( tm.extent().yMinimum(), -20037508.3427892, 0.0000001 ) );
   QVERIFY( qgsDoubleNear( tm.extent().xMaximum(), 20037508.3427892, 0.0000001 ) );
   QVERIFY( qgsDoubleNear( tm.extent().yMaximum(), 20037508.3427892, 0.0000001 ) );
-  QVERIFY( qgsDoubleNear( tm.scale(), 559082264.0287178, 0.0000001 ) );
+  QVERIFY( qgsDoubleNear( tm.scale(), 559082264.0287166, 0.0000001 ) );
```

Comment:
you could actually keep the original value 559082264.0287178 and just lower the tolerance - even a millimeter (0.001) should be more than enough :-) but I am easy...